hikari/impl/rest.py

```diff
@@ -503,6 +504,11 @@ class RESTClientImpl(rest_api.RESTClient):
 
         This is provided since some endpoints may respond with non-sensible
         rate limits.
+    max_retries : typing.Optional[builtins.int]
+        Maximum number of times a request will be retried if
+        it fails with a `5xx` status. Defaults to 3 if set to `builtins.None`.
+
+        If you provide above 5, it will default to 5.
```

Comment:
```suggestion
        If you provide a value above 5, it will default to 5.
```
Just a little wording issue